website/content/docs/v0.8/Cloud Platforms/aws.md

```diff
@@ -148,6 +146,15 @@ created join.yaml
 created talosconfig
 ```
 
+Take note that in this version of Talos, the generated configs are too long for AWS userdata field.
```

Comment:
idk if we need that, but since 0.10 we support `talosctl gen config --with-docs=false --with-examples=false`

(should be on 0.10 doc, but I couldn't find any good place for this comment)

Reply:
yeah, I at least made a note as to why those flags were required.